source/software/singularity.rst

```diff
@@ -172,30 +176,37 @@ be shared, and improve reproducibility.
 How can I run a Singularity image?
 ----------------------------------
 
-Once you have an image, there are several options to run it.
+Once you have an image, there are several options to run the container.
 
-#. You can invoke any application that is in the PATH of the image, e.g.,
-   for the image containing Grace::
+#. You can invoke any application that is in the ``$PATH`` of the
+   container, e.g., for the image containing Grace::
 
    $ singularity  exec  grace.sif  xmgrace
 
-#. In case the definition file specified a ``runscript``, this can be
-   executed using::
+#. In case the definition file specified a ``%runscript`` directive,
+   this can be executed using::
 
    $ singularity  run  grace.sif
 
-#. The image can be run as a shell::
+#. The container can be run as a shell::
 
    $ singularity  shell  grace.sif
 
-By default, your home directory will be mounted with the same path
-as it has on the host.  The current working directory is that on
-the host in which you invoked ``singularity``.
+By default, your home directory in the container will be mounted
+with the same path as it has on the host.  The current working
+directory in the container is that on the host in which you
+invoked ``singularity``.
+
+.. note::
 
-Additional host directories can be mounted in the image as well by
-using the ``-B`` option.  Mount points are created dynamically, so
-they do not have to exist in the image.  For example, to mount the
-``$VSC_SCRATCH`` directory, you would use::
+   Although you can move to a parent directory of the current working
+   directory in the container, you will not see its contents on the host.
+   Only the current working directory on the host was mounted.
```

Comment:
maybe replace with:
Only the current working directory and its sub-directories on the host are visible by default.

Reply:
Done.